setup.py

```diff
@@ -79,7 +79,7 @@ def run(self):
 
 install_requires = [
     'six>=1.7',
-    'nvidia-ml-py3>=7.352.0',
+    'nvidia-ml-py!=375.*',  # see https://forums.developer.nvidia.com/t/mistake-with-nvidia-ml-py-on-pypi/112942
```

Comment:
We should freeze this version as `nvidia-ml-py==11.450.51` before we can find a way to patch `nvmlProcessInfo_t` and `nvmlDeviceGetComputeRunningProcesses`.

Reply:
I don't like pinning the exact version because it will cause many other problems (e.g., an unexpected version still might be installed despite the pinned dependency specification).

Actually `nvidia-ml-py==11.460.79` (which is the latest and >=11.450.129) works fine for me. It is giving me a correct process result with the latest driver version (I tested with 470.57.02).

With old driver versions (e.g. 430.64), however, getting the function pointer to `nvmlDeviceGetComputeRunningProcesses_v2` will fail according to your report. I'll need to add a patch in this PR. Some details of how to elegantly do this is something we need to figure out.

Not sure what we need to patch `nvmlProcessInfo_t`. Is it because the struct size has changed?